tools/keepalived/lib/scheduler.c

```diff
@@ -1641,15 +1642,15 @@ thread_fetch_next_queue(thread_master_t *m)
 
 	assert(m != NULL);
 
-	/* If there is event process it first. */
```

Comment:
Please confirm, @acassen/keepalived#1631 just added "thread_events_resize" but did not change this loop.

Reply:
The change of loop is for optimization. 
For example, the checker child process runs in the loop of thread_fetch_next_queue， then we send HUP signal to keepalived for reloading, epoll_wait  will return EINVAL error, and then it will create an THREAD_EVENT type event by the Signal callback function of thread_add_event. The reload operation will not be executed immediately, and will not be executed until the loop ends and enters the function again;